.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+# Pre-commit (https://pre-commit.com) configuration for assorted lint checks.
+#
+# See https://pre-commit.com/hooks.html for more hooks.
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: check-merge-conflict
+
+  - repo: https://github.com/psf/black
+    rev: 23.3.0
+    hooks:
+      - id: black
+        name: Run Black to format Python files
+
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.41.0
+    hooks:
+      - id: markdownlint
+        name: Run markdownlint on .md files
+        args: ["--config", "docs/.markdownlint.yml"]
+        files: "docs/website/.*.md"
+        exclude: "mlir-dialects/!(index).md"
```

Comment:
Can remove this hook or edit the paths. There is no `docs/` folder in this repo.